cmd/kpod/create.go

```diff
@@ -0,0 +1,160 @@
+package main
+
+import (
+	"fmt"
+
+	spec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+	pb "k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime"
+)
+
+type createResourceConfig struct {
+	blkioWeight       int64
+	blkioDevice       []string
+	cpuShares         int64
+	cpuCount          int64
+	cpuPeriod         int64
+	cpusetCpus        string
+	cpusetNames       string
+	cpuFile           string
+	cpuMems           string
```

Comment:
What are the correlating command line options for this one and the two above it?

Reply:
Removing, leftovers.